src/main/java/net/earthcomputer/clientcommands/ClientCommands.java

```diff
@@ -161,6 +165,9 @@ public static void registerCommands(CommandDispatcher<FabricClientCommandSource>
         WeatherCommand.register(dispatcher);
         PluginsCommand.register(dispatcher);
         CGameModeCommand.register(dispatcher);
+        if (ListenCommand.isEnabled) {
```

Comment:
I've just thought of another race condition. This command could be registered and then the `isEnabled` boolean set to false afterwards. On Quilt, client-side commands are registered on startup as well for some reason that isn't clear to me. Might be a better idea to always register the command and check the boolean when it's executed.